### PR TITLE
{lib}[foss/2020b] igraph 0.8.5

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/igraph/igraph-0.8.5-GCC-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+dependencies = [
+    ('GLPK', '4.65'),
+    ('libxml2', '2.9.10'),
+    ('zlib', '1.2.11'),
+]
+
+preconfigopts = 'autoreconf -i && '
+# Remove hardcoded links to BLAS/LAPACK
+preconfigopts += "sed -i 's/-lblas/$LIBBLAS/' configure && "
+preconfigopts += "sed -i 's/-llapack/$LIBLAPACK/' configure && "
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/i/igraph/igraph-0.8.5-GCC-10.2.0.eb:33]: shouldn't this use the `foss` toolchain then?

Comment (fizwit) [reply easybuild/easyconfigs/i/igraph/igraph-0.8.5-GCC-10.2.0.eb:33]: my bad,

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u38b.bear.cluster - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/a3ca494b9c8e20e1311f7cdafb7d1b23 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/d75546e0518d0ffa21644f83b3cf0158 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @fizwit!

**Pull request merged**